internal/pkg/agent/cmd/setup_config.go

```diff
@@ -76,15 +78,16 @@ func defaultAccessConfig() (setupConfig, error) {
 
 	cfg := setupConfig{
 		Fleet: fleetConfig{
-			CA:              envWithDefault("", "FLEET_CA", "KIBANA_CA", "ELASTICSEARCH_CA"),
-			Enroll:          envBool("FLEET_ENROLL", "FLEET_SERVER_ENABLE"),
-			EnrollmentToken: envWithDefault("", "FLEET_ENROLLMENT_TOKEN"),
-			Force:           envBool("FLEET_FORCE"),
-			Insecure:        envBool("FLEET_INSECURE"),
-			TokenName:       envWithDefault("Default", "FLEET_TOKEN_NAME"),
-			TokenPolicyName: envWithDefault("", "FLEET_TOKEN_POLICY_NAME"),
-			URL:             envWithDefault("", "FLEET_URL"),
-			DaemonTimeout:   envTimeout("FLEET_DAEMON_TIMEOUT"),
+			CA:                     envWithDefault("", "FLEET_CA", "KIBANA_CA", "ELASTICSEARCH_CA"),
+			Enroll:                 envBool("FLEET_ENROLL", "FLEET_SERVER_ENABLE"),
+			EnrollmentToken:        envWithDefault("", "FLEET_ENROLLMENT_TOKEN"),
+			Force:                  envBool("FLEET_FORCE"),
+			Insecure:               envBool("FLEET_INSECURE"),
+			TokenName:              envWithDefault("Default", "FLEET_TOKEN_NAME"),
+			TokenPolicyName:        envWithDefault("", "FLEET_TOKEN_POLICY_NAME"),
+			DefaultTokenPolicyName: envWithDefault("Default policy", "DEFAULT_FLEET_TOKEN_POLICY_NAME"),
```

Comment:
(nitpick) Shouldn't the env var be `FLEET_DEFAULT_TOKEN_POLICY_NAME`? as all other env vars start with `FLEET_`?